### DISCUSSION TITLE
Run a podman container using system root as container filesystem root

### DISCUSSION THREAD

**Discussion by mildred** [containers/podman#17708] [Ideas]
Sometimes we need to run containers that needs to access the host filesystem as if it was running on the host, but the docker image is still a great way tu bundle things up.

It would be nice to be able to tell when running a container that the default mount points should all come from the host, and only a specific path within the root filesystem (creating directories as needed) would contain the filesystem of the container image. This is rarely necessary but some cases might require it.

Latest example I have is the prometheus node_exporter. There are also flatpaks that might have some use for this but as far as I know they do not use podman.

**Comment by rhatdan** [comment]
We attempted this type of thing years ago with the RUN_LABEL command.

https://docs.podman.io/en/latest/markdown/podman-container-runlabel.1.html

There is no other standard way to handle this, that I know of.

**Comment by mildred** [comment]
Thanks, I also discovered `podman image mount` which will get the files on host and then systemd can be used to isolate further the process executed.
